web/package.json

```diff
@@ -94,7 +96,7 @@
     "@types/jest": "^29.0.0",
     "@types/recharts": "^1.8.23",
     "cypress": "^12.0.0",
-    "eslint": "^8.12.0",
+    "eslint": "8.12.0",
```

Comment:
Is this an intended change?

Reply:
Gah, sorry, more npm woes. I've unpinned the version.